### ISSUE TITLE
Complete accessibility testing

### ISSUE THREAD

**Issue by DaveBathnes** [LibrariesHacked/schema-librarydata#11]
The site should be accessible and usable by everyone.

For public sector websites the minimum requirement is WCAG 2.1 AA. Given the website is mainly text, images and links, we should test it to that standard.

The accessibility page should then be updated with testing details and results

**Acceptance Criteria**
- Site tested using manual basic testing
- Accessibility page updated
